packages/SwingSet/test/liveslots-helpers.js

```diff
@@ -14,13 +14,19 @@ import {
   makeBringOutYourDead,
 } from './util.js';
 
-export function buildSyscall() {
+export function buildSyscall(skipLogging) {
```

Comment:
```suggestion
/**
  * @param {boolean} [skipLogging=false]
  */
export function buildSyscall(skipLogging) {
```

I'm surprised that this still returns `log` (even empty) when `skipLogging` is true. No request re: that.

Reply:
I was just being lazy, since this is not production code.  The point was simply to avoid accumulating a bunch of crap in a buffer that's going to be ignored anyway.  But in the interest of hygiene I can change the return to:
```
return skipLogging ? { syscall } : { log, syscall };
```
(I'll also add the param type comment)